From 732c473b0f25af458d84248d65ccfa9fd098b288 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Wed, 1 Dec 2021 20:51:58 +0000
Subject: [PATCH 436/462] MANTIS 0061964: [Touchless] Rest API Call when an
 application leaves the status "active"

add property JarTouchlessClient
add java module ${PlatformJavaDir}/TouchlessClient

git-svn-id: svn://localhost/SelfServiceBuild/trunk@829 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntProperties.xml | 15 ++++++++-------
 SelfServiceBuild/Build/java/build.xml         |  1 +
 2 files changed, 9 insertions(+), 7 deletions(-)

diff --git a/SelfServiceBuild/Build/java/AntProperties.xml b/SelfServiceBuild/Build/java/AntProperties.xml
index 852b579..61a5972 100644
--- a/SelfServiceBuild/Build/java/AntProperties.xml
+++ b/SelfServiceBuild/Build/java/AntProperties.xml
@@ -237,14 +237,14 @@
 <property name="JarCUSS"                      value="${JarDir}/cuss.jar"/>
 <property name="JarCLAGUI"                    value="${JarDir}/CLA_GUI.jar"/>
 <property name="JarCUSSPayment"               value="${JarDir}/CUSS.PAYMENT.XSD.jar"/>
-<property name="JarCUSSSBD"                   value="${JarDir}/CUSS.SBD.XSD.jar"/> 
+<property name="JarCUSSSBD"                   value="${JarDir}/CUSS.SBD.XSD.jar"/>
 <property name="JarCUSSIllumination"          value="${JarDir}/CUSS.ILLUMINATION.XSD.jar"/>
-<property name="JarCUSSBiometrics"            value="${JarDir}/CUSS.BIOMETRIC.XSD.jar"/> 
-<property name="JarCUSS15Biometrics"          value="${JarDir}/CUSS-1.5.BIOMETRICS.XSD.jar"/> 
-<property name="JarCUSS15BiometricsGP"        value="${JarDir}/CUSS-1.5.BIOMETRICS.GP.XSD.jar"/> 
-<property name="JarCUSSIdDocument"            value="${JarDir}/IdDocument.XSD.jar"/> 
-<property name="JarCUSSSynthesis"             value="${JarDir}/Synthesis.XSD.jar"/> 
-<property name="JarToolsData"                 value="${JarDir}/ToolsData.jar"/> 
+<property name="JarCUSSBiometrics"            value="${JarDir}/CUSS.BIOMETRIC.XSD.jar"/>
+<property name="JarCUSS15Biometrics"          value="${JarDir}/CUSS-1.5.BIOMETRICS.XSD.jar"/>
+<property name="JarCUSS15BiometricsGP"        value="${JarDir}/CUSS-1.5.BIOMETRICS.GP.XSD.jar"/>
+<property name="JarCUSSIdDocument"            value="${JarDir}/IdDocument.XSD.jar"/>
+<property name="JarCUSSSynthesis"             value="${JarDir}/Synthesis.XSD.jar"/>
+<property name="JarToolsData"                 value="${JarDir}/ToolsData.jar"/>
 <property name="JarMassaiConfig"              value="${JarDir}/massai-config.jar"/>
 <property name="JarMassaiControlCenter"       value="${JarDir}/massai-controlcenter.jar"/>
 <property name="JarMassaiControlCenterGate"   value="${JarDir}/massai-controlcenter-gate.jar"/>
@@ -284,6 +284,7 @@
 <property name="JarKioskSelfTestSchema"       value="${JarDir}/mips-kioskSelfTestSchema.jar"/>
 <property name="JarServiceManager"            value="${JarDir}/mips-ServiceManager.jar"/>
 <property name="JarServiceManagerSchema"      value="${JarDir}/mips-ServiceManagerSchema.jar"/>
+<property name="JarTouchlessClient"           value="${JarDir}/mips-TouchlessClient.jar"/>
 
 
 <!-- slf4j -->
diff --git a/SelfServiceBuild/Build/java/build.xml b/SelfServiceBuild/Build/java/build.xml
index 56a85f5..d723b33 100644
--- a/SelfServiceBuild/Build/java/build.xml
+++ b/SelfServiceBuild/Build/java/build.xml
@@ -60,6 +60,7 @@
       ${PlatformJavaDir}/CLAApplet,
       ${PlatformJavaDir}/DSGEventReporting,
       ${PlatformJavaDir}/ExtractBilling,
+      ${PlatformJavaDir}/TouchlessClient,
       ${PlatformJavaDir}/BhsServer"
     />
     <property name="AblsDirList"     value="${AblsJavaDir}/KioskProxy,${AblsJavaDir}/TunnelServer"/>
-- 
2.41.0.windows.1

